test/checkbox.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { test, Page } from "bigtest";
+import { Body, CheckBox as Interactor } from "../src/index";
+import { Checkbox, FormControlLabel } from "@material-ui/core";
+import { render } from "./helpers";
+
+export default test("Checkbox")
+  .step(Page.visit("/"))
+  .child("filter", (test) =>
+    test
+      .child("by locator", (test) =>
+        test
+          .step("render", () => render(<FormControlLabel label="locator" control={<Checkbox />} />))
+          .assertion(Interactor("locator").exists())
```

Comment:
Is this test to check for something around the label? Because i think the assertion here is testing the existence of the button.